pkg/provider/handlers/functions.go

```diff
@@ -49,15 +51,19 @@ func GetFunction(client *containerd.Client, name string) (Function, error) {
 
 		containerName := c.ID()
 		labels, labelErr := c.Labels(ctx)
-		if labelErr != nil {
+                // TODO Here need to retrieve the special list of labels that are actually
```

Comment:
See for example: https://github.com/openfaas/faas-swarm/blob/ef7d9132e29edc8ff3962685ba772ad6e3f01bda/handlers/reader.go#L57